tasks.py

```diff
@@ -873,3 +868,22 @@ def deployprometheus(ctx):
     print("Waiting for prometheus pods to be running")
     run("kubectl -n monitoring wait --for=condition=Ready --all pods --timeout 300s")
 
+def _fetch_kubectl():
+    if not os.path.exists(build_path):
+        os.makedirs(build_path, mode=0o750)
+    curl_command = "curl -o {} -LO https://dl.k8s.io/release/{}/bin/$(go env GOOS)/$(go env GOARCH)/kubectl".format(kubectl_path, kubectl_version)
+    if not os.path.exists(kubectl_path):
+        run(curl_command)
+        run("chmod +x {}".format(kubectl_path))
+        return
+    version = run("{} version --short".format(kubectl_path), warn=True).stdout
+    for line in version.splitlines():
+        if line.startswith("Client Version:"):
+            version = line.split(":")[1].strip()
+            if version == kubectl_version:
+                return
+    run(curl_command)
+    run("chmod +x {}".format(kubectl_path))
+
+_fetch_kubectl()
```

Comment:
why are you calling this function here? I see that other helper functions are called where needed. To keep consistency, I would expect this to be called inside dev_env itself.

Reply:
because I don't want to bloat every function with a new line, as opposed to check the version every time we call env (and not doing anything if there is nothing to do)

Reply:
as a second thought, let me recheck how many functions are calling it

Reply:
changed